packages/idos-store/index.ts

```diff
@@ -6,6 +6,30 @@ interface PipeCodecArgs<T> {
 export class Store {
   keyPrefix = "idOS-";
 
+  constructor() {
+    for (const key in localStorage) {
+      if (key.startsWith(this.keyPrefix) && key.endsWith("-expires")) {
+        const value = localStorage.getItem(key);
+        if (value) {
+          let str;
+
+          try {
+            str = JSON.parse(value);
+          } catch (error) {
+            str = "";
```

Comment:
Do we want to do anything else if we can't decode?
```suggestion
            continue;
```